sunholmexp.py

```diff
@@ -148,10 +149,15 @@ def main() -> None:
     parser_new_player.add_argument('playername', type=str, help="The name of the new player.")
     parser_new_player.add_argument('--startingxp', metavar="xp", type=int, help="How much XP should the player start with", default=300)
 
-    parser_bonus_exp = subparsers.add_parser("bonus", help="Give an out of band bonus to a player")
+    parser_bonus_exp = subparsers.add_parser("bonus", help="Give an out of band xp bonus to a player")
     parser_bonus_exp.add_argument('playername', type=str, help="The name of the player.")
     parser_bonus_exp.add_argument('bonusxp', type=int, help="How much xp bonus to give.")
 
+    parser_levelup = subparsers.add_parser("levelup", help="Give an out of band level bonus to a player")
+    parser_levelup.add_argument('playername', type=str, help="The name of the player.")
+    parser_levelup.add_argument('levels', type=int, help="How many bonus levels to give.")
+    parser_levelup.add_argument('preserve', type=bool, help="Preserve level progress as a percentage.")
```

Comment:
lets change this to:
```
parser_levelup.add_argument('--preserve-percentage', help="Preserve level progress as a percentage.", action='store_true')
```
I like having verbosity when something  is being used, eg calling it preserve-percentage to better illustrate what it is doing. I also like removing unnecessary verbosity, eg: having to type `false` at the end of any levelup command.

This will also change the call to the variable below to be `parsed_args.preserve_percentage`